model-engine/model_engine_server/api/model_endpoints_v1.py

```diff
@@ -216,3 +243,10 @@ async def delete_model_endpoint(
             status_code=500,
             detail="deletion of endpoint failed, compute resources still exist.",
         ) from exc
+    except Exception as exc:
+        request_id = get_request_id()
```

Comment:
Think this is fine for now, but as we revisit self-hosting in VPCs, we'll probably want some sort of Gateway abstraction for getting the request/trace ID.